Applications/Spire/Source/KeyBindings/OrderFieldAdditionalTag.cpp

```diff
@@ -0,0 +1,92 @@
+#include "Spire/KeyBindings/OrderFieldAdditionalTag.hpp"
+#include "Spire/Spire/ArrayListModel.hpp"
+#include "Spire/Ui/CalendarDatePicker.hpp"
+#include "Spire/Ui/DateBox.hpp"
+#include "Spire/Ui/DecimalBox.hpp"
+#include "Spire/Ui/DropDownBox.hpp"
+#include "Spire/Ui/DurationBox.hpp"
+#include "Spire/Ui/IntegerBox.hpp"
+#include "Spire/Ui/MoneyBox.hpp"
+#include "Spire/Ui/QuantityBox.hpp"
+#include "Spire/Ui/TextBox.hpp"
+
+using namespace boost;
+using namespace boost::gregorian;
+using namespace boost::posix_time;
+using namespace Nexus;
+using namespace Spire;
+
+namespace {
+  auto make_quantity_modifiers() {
+    return QHash<Qt::KeyboardModifier, Quantity>(
```

Comment:
Why is this constantly repeated everywhere in the codebase?

Reply:
Okay sounds good. Create a task for this issue and implement it in a separate branch.